README.md

```diff
@@ -30,7 +67,17 @@ Currently the following variables are supported:
 Dependencies
 ------------
 
-None
+The adm system needs to already be configured with [subscription-manager](https://github.com/oasis-roles/rhsm)
+to include the necessary repositories as described in the documentation. Additionally the local
+Ansible inventory and group\_vars folders need to already exist on the local system so that they
+can be uploaded to the remote target. This role will install the "openshift-ansible" package, so
+whichever repositories provide the particular version of that package to correspond with the version
+of OCP you are trying to deploy.
+
+Additionally, you need to have a local copy of the SSH private key file that will be used to
+log in to the remote hosts, if these are not explicitly defined elsewhere in group\_vars and
+the inventory file. Obviously, if other methods of authentication are in use, then those are
+outside of the scope of this playbook.
```

Comment:
`s/playbook/role`? Other auth methods are presumably outside the scope of this role, but probably still inside the scope of a playbook using this role.

Reply:
Good call